learing_journal/scripts/initializedb.py

```diff
@@ -32,8 +33,9 @@ def main(argv=sys.argv):
     options = parse_vars(argv[2:])
     setup_logging(config_uri)
     settings = get_appsettings(config_uri, options=options)
-
+    # settings['sqlalchemy.url'] = os.environ['DATABASE_URL']
```

Comment:
[Step 4] This shouldn't be commented. You should indeed replace the "sqlalchemy.url" keyword with something from the environment.